mdr/mdr.py

```diff
@@ -73,7 +90,31 @@ def transform(self, features):
             Constructed features from the provided feature matrix
 
         """
-        return 0
+        new_feature = np.zeros(shape=(features.shape[0],1), dtype=np.int)
```

Comment:
Transform should be much shorter due to the changes above. It should only go through each record in `features` and look up the tuple in `self.feature_map`. The array that is returns should be of the same length as the `features` array.